habitat/tasks/nav/nav_task.py

```diff
@@ -75,9 +75,11 @@ class ObjectGoal(NavigationGoal):
 class RoomGoal(NavigationGoal):
     r"""Room goal that can be specified by room_id or position with radius.
     """
-
-    room_id: str = attr.ib(default=None, validator=not_none_validator)
-    room_name: Optional[str] = None
+    room_aabb: Tuple[float] = attr.ib(
```

Comment:
Let's move all RoomNav Task related definitions to `habitat/tasks/nav/room_nav_task.py` and include it here. This Nav task file already grew too much and is hard to read.